AUTHORS

```diff
@@ -3611,5 +3611,14 @@ Nicolas DUBIEN <github@dubien.org>
 richiemccoll <richie.mccoll@hotmail.co.uk>
 sinkhaha <1468709106@qq.com>
 Mert Can Altın <mertgold60@gmail.com>
+Jan Osusky <Jan.Osusky@iblsoft.com>
+Gabriela Gutierrez <gabigutierrez@google.com>
+Julian Dropmann <julian@dropmann.org>
+Steve Herzog <srherzog@gmail.com>
+Abdirahim Musse <33973272+abmusse@users.noreply.github.com>
+Andreas Martens <amartens@uk.ibm.com>
+Ramana Venkata <vramana@users.noreply.github.com>
+jakecastelli <38635403+jakecastelli@users.noreply.github.com>
+Vita Batrla <34657903+batrla@users.noreply.github.com>
```

Comment:
Duplicate: Both entries in AUTHORS for Vita Batrla are for @batrla. Needs a .mailmap entry to consolidate.